### DISCUSSION TITLE
Visual Studio support

### DISCUSSION THREAD

**Discussion by mterwoord** [qwertie/ecsharp#38] [Q&A]
You have any plans to support Visual Studio? (Intellisense, etc)

(Also, bonus points for resharper support)

**Comment by qwertie** [comment]
It has always been my goal to target VS as the main IDE (LeMP is a [Visual Studio extension](https://github.com/qwertie/ecsharp/releases) already), and I'd love to get IntelliSense and all that. Unfortunately, development has been slow: partly because I'm working on something for WebAssembly, partly for personal reasons, and partly because there is only one other developer and he's on Linux.

**Comment by mterwoord** [comment]
Well, maybe focusing on Visual Studio Code is an option then? THat's cross platform. (And I have high hopes for normal visual studio to support the same interface at some point in the future)

**Comment by qwertie** [comment]
Hmm, I still haven't gotten around to making a Visual Studio Code extension, mainly because of the hassle of figuring out how to use the language used by Visual Studio to represent syntax highlighters... something called "TextMate grammars". And IntelliSense remains impractical. Having a budget of zero isn't recipe for rapid progress, I'm afraid. But at least now it supports running arbitrary C# code and custom macros at compile time. And I'm working on a web-based Blazor version.
